config/locales/en.yml

```diff
@@ -78,6 +78,11 @@ en:
       show_places: 'Show places in my area'
     show:
       users_of_events: 'List of users of this event:'
+    edit:
+      edit_event: 'Editing Event'
+      links: 
+        show: 'Show'
```

Comment:
You add one more time the translations to links which where here before. Please do not duplicate translations.

Reply:
still relevant